examples/c/example.c

```diff
@@ -384,11 +384,33 @@ AranyaError init_team(Team* t) {
     // real world scenario, the keys would be exchanged outside of Aranya using
     // something like `scp`.
 
+    size_t owner_role_len = 1;
+    AranyaRole owner_role[owner_role_len];
```

Comment:
Would prefer to call this `team_roles` then set `Role owner_role = team_roles[0]` after checking that only 1 role was returned.
```suggestion
    AranyaRole team_roles[owner_role_len];
```